I Love Music/dist/presence.js

```diff
@@ -0,0 +1,90 @@
+//import { write } from "fs";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+var presence = new Presence({
+    clientId: "477919120789078026",
+    mediaKeys: false
+}), strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+var title, views, air, air2, title2;
+var iFrameVideo, currentTime, duration, paused;
+var video, videoDuration, videoCurrentTime;
+var lastPlaybackState = null;
+var playback;
+var browsingStamp = Math.floor(Date.now() / 1000);
+if (lastPlaybackState != playback) {
+    lastPlaybackState = playback;
+    browsingStamp = Math.floor(Date.now() / 1000);
+}
+
+presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
+    var a = presenceData = '', presenceData = {
+        largeImageKey: "logo"  
```

Comment:
var a = presenceData = '',  can be removed lmao

Reply:
`presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
    var a = presenceData = {
        largeImageKey: "logo"  
    };`
If I remove this I get an error

Reply:
https://docs.premid.app/en/dev/presence#getting-started-1

Reply:
Thank you x) 
I removed this `a = `